server/src/pages/shot/controller.js

```diff
@@ -209,6 +214,7 @@ exports.setTitle = function(title) {
     if (req.status >= 300) {
       let errorMessage = document.getElementById("shotPageAlertErrorUpdatingTitle").textContent;
       window.alert(errorMessage);
+      window.Raven.captureException(new Error(`${errorMessage}: ${req.status} ${req.statusText}`));
```

Comment:
These should use something simpler than `${errorMessage}` – that's going to be localized, which will confuse things in Sentry.  Something like `Error calling /api/set-title: ${req.status} ${req.statusText}`